pandas/tseries/tests/test_resample.py

```diff
@@ -2043,17 +2045,30 @@ def test_resample_basic(self):
         assert_series_equal(result2, expected)
 
     def test_resample_empty(self):
-
         # GH12771 & GH12868
         index = PeriodIndex(start='2000', periods=0, freq='D', name='idx')
         s = Series(index=index)
 
-        expected_index = PeriodIndex([], name='idx', freq='M')
-        expected = Series(index=expected_index)
-
-        for method in resample_methods:
-            result = getattr(s.resample('M'), method)()
-            assert_series_equal(result, expected)
+        for freq in ['M', 'D', 'H']:
+            for method in (resample_methods - {'ohlc'}):
```

Comment:
just make a new test method for this. ``resample_no_ohlc_methods``; the ordering is very important for debugging.